.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
Please remove all the `.vs` files from the PR. You can add `.vs` to your personal `.gitignore`, or make another PR adding it to kytos' project-level `.gitignore`.